src/azure-cli/azure/cli/command_modules/aro/_aad.py

```diff
@@ -19,7 +19,7 @@
 logger = get_logger(__name__)
 
 
-class AADManager:
+class AADManager():
```

Comment:
@bennerv I'd expect this to be `class AADManager(object):`

Reply:
@jim-minter `azdev style aro` complains about it when it's there.  I can remove the `()` if we think that's best so it shows no change.

```
************* Module azure.cli.command_modules.aro._aad
_aad.py:22:0: R0205: Class 'AADManager' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
```